### ISSUE TITLE
Can you share the segmentation code?

### ISSUE THREAD

**Issue by mongeoroo** [KAIST-vilab/AEFT#3]
I tried learning deeplab model with provided pretrained weight. (I tried deeplab V1 and B2.)

It achieved 71% on the validation set with PSA.

But I can't get the final performance with segmentation model.

Can you share your segmentation code to reproduce the paper's performance?

**Comment by yoon307** [comment]
We plan to incorporate the codes (iccv (OC-CSE), ECCV (AEFT), and new paper that will be presented in CVPR23) with the segmentation code. But it may take some time to organize. To implement by yourself, refer segmentation code (ResNet38) of MCTformer (CVPR22) + multi scale training (MSC) code (https://github.com/kazuto1011/deeplab-pytorch/blob/master/libs/models/msc.py).
